domains/0v90.json

```diff
@@ -0,0 +1,17 @@
+{
+  "owner": {
+    "username": "0v90",
+    "email": "0x0jx@proton.me"
+  }
+}
+
+
+{
+  "record": {
+    "A": [
+      "35.200.171.190",
+      "35.240.84.96",
+      "34.132.146.171"
```

Comment:
Since these are just saying "webserver is functioning normally", I'd like you to give a preview of your website